app/js/controllers/gesture.js

```diff
@@ -0,0 +1,79 @@
+/*global Controller*/
+
+export default class GestureController extends Controller {
+  constructor(options) {
+    super(options);
+
+    var firstTouchStartEvt = null;
+    var isForwarding = false;
+
+    window.addEventListener('touchstart', (evt) => {
+      if (evt.touches.length === 1) {
+        firstTouchStartEvt = evt;
+        return;
+      }
+
+      if (evt.touches.length !== 2) {
```

Comment:
Yeah, I debated including this, but I wanted to keep it as restrictive as possible for the gesture we are using.